src/en/wip-storage.md

```diff
@@ -31,19 +31,21 @@ juju storage volume list
 
 As previously mentioned, a charm which requires storage will automatically allocate the default storage (unit filesystem) by default. It is possible to instead specify the storage to be used when the service is deployed, using constraints.
 
-The constraints can specify the type/pool and size of the storage required.
+The constraints can specify the type/pool and size, and optionally a count, of the storage required.
```

Comment:
From what I have seen, type/pool and size have reasonable defaults as well. So, they could be "optionally" specified as well :)